README.md

```diff
@@ -1,6 +1,6 @@
 # remote_syslog2
 
-[![Download remote_syslog2](http://papertrail.github.io/remote_syslog2/images/download.png)][releases]
+[![Download remote_syslog2](https://papertrail.github.io/remote_syslog2/images/download.png)][releases]
```

Comment:
Changed all `http://` links to `https://` while I was in here. Confirmed that the new links all work.